packages/teams-js/src/public/people.ts

```diff
@@ -48,10 +48,8 @@ export namespace people {
   ): Promise<PeoplePickerResult[]> {
     ensureInitialized(runtime, FrameContexts.content, FrameContexts.task, FrameContexts.settings);
 
-    /* eslint-disable-next-line strict-null-checks/all */ /* Fix tracked by 5730662 */
-    let callback: selectPeopleCallbackFunctionType;
-    /* eslint-disable-next-line strict-null-checks/all */ /* Fix tracked by 5730662 */
-    let peoplePickerInputs: PeoplePickerInputs;
+    let callback: selectPeopleCallbackFunctionType | undefined = undefined;
```

Comment:
>| undefined [](http://example.com/codeflow?start=51&length=11)

These two lines were previously always set to `undefined` initially by not assigning a value explicitly.